util/gh-pages/index.html

```diff
@@ -216,40 +233,31 @@ <h4 class="list-group-item-heading">
             };
 
             $scope.bySearch = function (lint, index, array) {
-                let search_str = $scope.search;
+                let searchStr = $scope.search;
                 // It can be `null` I haven't missed this value 
-                if (search_str == null || search_str.length == 0) {
+                if (searchStr == null || searchStr.length < 3) {
                     return true;
                 }
-                search_str = search_str.toLowerCase();
+                searchStr = searchStr.toLowerCase();
 
                 // Search by id
-                let id_search = search_str.trim().replace(/(\-| )/g, "_");
-                if (lint.id.includes(id_search)) {
+                if (lint.id.indexOf(searchStr.replace("-", "_")) !== -1) {
```

Comment:
The previous code also replaced spaces with underscores. Not sure if this is still desired but I at least wanted to point it out :)

Reply:
Jup this is on purpose. The space search is now covered by line 252 `if (lint.id.indexOf(therms[index]) !== -1)` this allows the words to be out of order which is likely when somebody just types them in. It could also reduce performance since we are not using a regex in the replace function but I'm not sure about that :)

Reply:
Ah of course, that makes sense!

(I would expect any decent JS enngine to optimize the simple regex to a byte match so it's probably as fast as the literal replace.)